sdk/metricsadvisor/ai-metrics-advisor/README.md

```diff
@@ -210,14 +210,15 @@ async function createDataFeed(adminClient, sqlServerConnectionString, sqlServerQ
   };
 
   console.log("Creating Datafeed...");
-  const result = await adminClient.createDataFeed({
+  const dataFeed = {
```

Comment:
We also want to get rid of the individual objects: source, granularity, dataFeedSchema, dataFeedIngestion, options, and write then inline inside of the dataFeed object literal

Reply:
done